spec/forms/candidate_interface/contact_details_form_spec.rb

```diff
@@ -112,6 +112,25 @@
     end
   end
 
+  describe '#sanitize_phone_number' do
+    it 'returns sanitized phone number' do
+      form_data = { phone_number: '123456789123456-+() ' }
+      phone_number = described_class.new(form_data)
+
+      expect(phone_number.sanitize_phone_number).to eq('123456789123456')
+    end
+  end
+
+  describe '#reset_phone_number' do
+    it 'returns raw phone number from user' do
+      form_data = { phone_number: '123456789123456', raw_phone_number: '123456789123456-+()' }
+      phone_numbers = described_class.new(form_data)
+      phone_numbers.reset_phone_number
+
+      expect(phone_numbers.phone_number).to eq(phone_numbers.raw_phone_number)
+    end
+  end
```

Comment:
Move these tests to the `phone_number_validator_spec.rb`?  Feels wrong to be testing methods called via callbacks – they should probably be private?

Reply:
We've re-written these to be unit tests for `phone_number_validator_spec.rb`